### ISSUE TITLE
🔗 🚀 Feature Request: Attachments only by email. No FTP (Duplicate of #4868)

### ISSUE THREAD

**Issue by lauranunez** [elementor/elementor#6167]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [ ] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
The attachements from form Elementor Pro now are in my email. I can see a code in this forum, but I need remove files from FTP. Only email. No server.

I received too emails attachements and I don´t want received in my FTP

**Describe the solution you'd like**
Only email attachments

**Comment by bainternet** [comment]
@lauranunez 	

The uploaded files are uploaded to the server, not FTP (FTP is the protocol and client you use to access files on your server).

now if you are asking about this snippet: 
```PHP
<?php
/**
 * Class Elementor_Form_Email_Attachments
 *
 * Send Elementor Form upload field as attachments to email
 */
class Elementor_Form_Email_Attachments {
	public $attachments_array = [];

	public function __construct() {
		add_action( 'elementor_pro/forms/process', [ $this, 'init_form_email_attachments' ], 11, 2 );
	}

	/**
	 * @param \ElementorPro\Modules\Forms\Classes\Form_Record $record
	 * @param \ElementorPro\Modules\Forms\Classes\Ajax_Handler $ajax_handler
	 */
	public function init_form_email_attachments( $record, $ajax_handler ) {
		// check if we have attachments
		$files = $record->get( 'files' );
		if ( empty( $files ) ) {
			return;
		}
		// Store attachment in local var
		foreach ( $files as $id => $files_array ) {
			$this->attachments_array[] = $files_array['path'][0];
		}

		// if local var has attachments setup filter hook
		if ( 0 < count( $this->attachments_array ) ) {
			add_filter( 'wp_mail', [ $this, 'wp_mail' ] );
			add_action( 'elementor_pro/forms/new_record', [ $this, 'remove_wp_mail_filter' ], 5 );
		}
	}

	public function remove_wp_mail_filter() {
		$this->attachments_array = [];
		remove_filter( 'wp_mail', [ $this, 'wp_mail' ] );
	}

	public function wp_mail( $args ) {
		$args['attachments'] = $this->attachments_array;
		return $args;
	}
}
new Elementor_Form_Email_Attachments();
```

then you can try to delete the files from the server after the mail is sent by replacing: 

```PHP
	public function remove_wp_mail_filter() {
		$this->attachments_array = [];
		remove_filter( 'wp_mail', [ $this, 'wp_mail' ] );
	}
```

with this:
```PHP
	public function remove_wp_mail_filter() {
		foreach( $this->attachments_array as $uploaded_file ) {
				unlink( $uploaded_file );
		}
		$this->attachments_array = [];
		remove_filter( 'wp_mail', [ $this, 'wp_mail' ] );
	}
```

Now this is not tested but its an example.

**Comment by bainternet** [comment]
ref #4868

**Comment by nicholaszein** [comment]
Hello there @lauranunez!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.17.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
